### ISSUE TITLE
"Incorrect line wrapping" even after fixing

### ISSUE THREAD

**Issue by kgsherman** [schoero/eslint-plugin-readable-tailwind#59]
Errors/warnings persist even after successfully fixing.
### Screenshot
![image](https://github.com/user-attachments/assets/cc1ab1ba-c521-4196-a1b8-cd027ad47d82)

### Package Versions
- eslint: 9.11.1
- typescript-eslint: 8.8.0
- eslint-plugin-readable-tailwind: 1.8.0

### eslint.config.js
```js
import eslint from "@eslint/js";
import tseslint from "typescript-eslint";
import solid from "eslint-plugin-solid/configs/typescript";
import eslintPluginReadableTailwind from "eslint-plugin-readable-tailwind";

export default [
  eslint.configs.recommended,
  {
    files: ["**/*.{ts,tsx}"],
    ...solid,
    languageOptions: {
      parser: tseslint.parser,
      parserOptions: {
        project: ["./tsconfig.json"],
      },
    },
    plugins: {
      "@typescript-eslint": tseslint.plugin,
    },
    rules: {
      ...tseslint.configs.recommendedTypeChecked.rules,
      "@typescript-eslint/consistent-type-imports": [
        "warn",
        {
          prefer: "type-imports",
          fixStyle: "separate-type-imports",
          disallowTypeAnnotations: true,
        },
      ],
    },
  },
  {
    files: ["**/*.{jsx,tsx}"],
    languageOptions: {
      parserOptions: {
        ecmaFeatures: {
          jsx: true,
        },
      },
    },
    plugins: {
      "readable-tailwind": eslintPluginReadableTailwind,
    },
    rules: {
      ...eslintPluginReadableTailwind.configs.warning.rules,
    },
  },
];
```

**Comment by schoero** [comment]
Unfortunately I was unable to reproduce the issue.

But I have released a new version, where the linting error messages/warnings are more verbose. They now show the current string with the fixed string and render invisible characters like whitespace and newlines.

Can you update to the new version and report back such a linting error with the new output? The output should look something like this:

```
Incorrect line wrapping. Expected

`inline-flex·items-center·justify-center·rounded-d·text-sm·font-medium↵
······transition-[color,background-color,box-shadow]↵
······disabled:pointer-events-none·disabled:opacity-50·focus-visible:outline-none·focus-visible:ring-[1.5px]↵
······focus-visible:ring-ring↵
····`

to be

`↵
······inline-flex·items-center·justify-center·rounded-d·text-sm·font-medium↵
······transition-[color,background-color,box-shadow]↵
↵
······disabled:pointer-events-none·disabled:opacity-50↵
↵
······focus-visible:outline-none·focus-visible:ring-[1.5px]·focus-visible:ring-ring↵
····`
```

I hope we can identify the issue that way.

**Comment by kgsherman** [comment]
Here's the updated error:
```
Incorrect line wrapping. Expected

`↩
↵
··········group·flex·grow·items-center·gap-4·text-nowrap·px-6·py-1.5↩
↵
↩
↵
··········md:text-lg↩
↵
········`

to be

`↵
··········group·flex·grow·items-center·gap-4·text-nowrap·px-6·py-1.5↵
↵
··········md:text-lg↵
········`eslint[readable-tailwind/multiline](https://github.com/schoero/eslint-plugin-readable-tailwind/blob/main/docs/rules/multiline.md)
```

**Comment by kgsherman** [comment]
Looks like the issue is `\n` vs `\n\r`. Switching VSCode to `LF` (rather than `CRLF`) solves this issue. I'll leave it to you whether or not it's worth an update to address!

**Comment by schoero** [comment]
If you rather want to use `\r\n` you can keep VSCode at `CRLF` and set `lineBreakStyle` to `"windows"` in the [multiline rule settings](https://github.com/schoero/eslint-plugin-readable-tailwind/blob/main/docs/rules/multiline.md)

```js
{
  rules: {
    ...eslintPluginReadableTailwind.configs.warning.rules,
    "readable-tailwind/multiline": ["warn", { lineBreakStyle: "windows" }],
  }
}
```

The updated output should make it more obvious what the error is, so I'm going to close this issue.
